src/commonMain/kotlin/exchange.dydx.abacus/state/v2/manager/StateManagerAdaptorV2.kt

```diff
@@ -707,5 +717,10 @@ internal class StateManagerAdaptorV2(
                 ),
             )
         }
+        triggerCompliance(ComplianceAction.CONNECT, null)
+    }
+
+    companion object {
+        private const val GEO_POLLING_DURATION_SECONDS = 10.0
```

Comment:
[q] why a private val instead of a global const?

Reply:
could probably make this a global const 👍